cmd/verrazzano-operator/main.go

```diff
@@ -50,14 +55,11 @@ func homePage(w http.ResponseWriter, r *http.Request) {
 // handleRequests registers the routes and handlers
 func handleRequests(apiServerFinished chan bool, rootRouter *mux.Router) {
 	apiRouter := registerPathHandlers(rootRouter)
-
-	glog.Info("Starting API server...")
-
+	zap.S().Info("Starting API server...")
 	if apiServerRealm == "" {
-		glog.Warning("Bearer token verification is disabled as apiServerRealm is not specified")
+		zap.S().Warn("Bearer token verification is disabled as apiServerRealm is not specified")
 	}
-
-	glog.Fatal(http.ListenAndServe(":3456", apiserver.CORSHandler(apiserver.AuthHandler(apiRouter))))
+	zap.S().Fatal(http.ListenAndServe(":3456", apiserver.CORSHandler(apiserver.AuthHandler(apiRouter))))
```

Comment:
Fatal is just a log level right? Does it have any impact on the running process?

Reply:
Fatal uses fmt.Sprint to construct and log a message, then calls os.Exit similar to the original glog.Fatal